addons/TrueFalse/src/presenter.js

```diff
@@ -516,6 +529,12 @@ function AddonTrueFalse_create() {
             }
             i++;
         });
+
+        if(parsedState.isDisabled && parsedState.isDisabled !== undefined) {
```

Comment:
ten if nie wymaga drugiej części. Lepiej tutaj zapisać if (parsedState.isDisabled === undefined) {
    parsedState.isDisabled = false;    // For backward compatibility in old lessons
}

if (parsedState.isDisabled) {

}...